packages/language-server/test/documentation-spec.ts

```diff
@@ -65,19 +57,5 @@ describe("The @ui5-language-assistant/language-server <getNodeDocumentation> fun
       expect(result.value).to.include("Experimental.");
       expect(result.value).to.not.include("since");
     });
```

Comment:
I think the test you removed was supposed to be without `since` and with `text` (or with both of them)